### DISCUSSION TITLE
Should we use a standard for the Building address or not?

### DISCUSSION THREAD

**Discussion by balazsdukai** [cityjson/specs#137] [Ideas]
This has been discussed in https://github.com/cityjson/specs/issues/51 already, and back then we chose for free-form addresses. I still think it was a good decision and the points discussed in the issue are still valid.

However, during the OGC Architecture Board meeting for the v1.1 revision acceptance, the question about the non-standard addresses came up. The critique was the classic "if it's not a standard then it's not interoperable" argument.
Someone recommended the IETF RFC-s, namely https://datatracker.ietf.org/doc/html/rfc5139 and https://datatracker.ietf.org/doc/rfc6848/ , as what we could use for the address.

So I want to float this idea again. Although, we discarded xAL in the past, maybe those RFC-s worth another look?

**Comment by kenohori** [comment]
These seem better, but they also seem overly complex IMO. Is there really a need for interoperability here? or is it just the OGC being OGC?

**Reply by balazsdukai** [reply]
Don't know to be honest, but it is probably the second.

**Comment by hugoledoux** [comment]
I propose we use what3words, I wonder what the OGC would think of this

**Comment by clausnagel** [comment]
Why not offer both? A set of predefined attributes for structured address information, and an additional string attribute for a free format address representation. I haven't checked the IETF links above but xAL, for instance, also offers this flexibility.
